### DISCUSSION TITLE
Icon

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#6522] [Help] [answered]
How to delete this icon 

![Capture](https://user-images.githubusercontent.com/106697208/176915220-8d7849da-8d53-4cda-8f63-3ac12f3f4c42.PNG)

**Comment by melohagan** [comment, marked as answer]
Hi @rafikApp 

The apps link is part of the navigation bar. You can hide the entire nav bar by going to *Layouts* -> *Navigation* = None
![Screenshot 2022-07-04 at 08 01 02](https://user-images.githubusercontent.com/101575380/177099399-d129ba95-3407-45db-8824-922870e7f4e2.png)

Alternatively, you can add an Embed to the layout with the following content:
```
<style>
.portal {
  visibility: hidden !important;
}
</style>
```

**Comment by rafikApp** [comment]
Hi @melohagan 
Thank you it works
